deploy/helm/charts/loki-rule-operator/Chart.yaml

```diff
@@ -15,10 +15,10 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: "0.7.4"
+version: "0.7.5"
 
 # This is the version number of the application being deployed. This version number should be
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "0.6.2"
```

Comment:
The image.tag is [set in the values.yaml](https://github.com/quero-edu/loki-rule-operator/blob/main/deploy/helm/charts/loki-rule-operator/values.yaml#L4). Can you remove it from the values? We will use the appVersion to set the default image tag for now on.

That done, I will merge this PR